asdf_astropy/converters/fits/tests/test_fitswcs.py

```diff
@@ -0,0 +1,68 @@
+import asdf
+import numpy as np
+import pytest
+from astropy import wcs
+
+
+def create_sip_distortion_and_tabular_wcs():
+    rng = np.random.default_rng()
```

Comment:
```suggestion
    rng = np.random.default_rng(42)
```
To make the tests deterministic.